apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -18,6 +21,20 @@ interface SectionsData {
   };
 }
 
+function getPathName(url: string): string {
+  const match = url.match(/\/\d+\/([^\/?]+)/);
+  const urlPath = match ? match[1] : '';
+
+  // Since the keys are different than the values (ex. lessonMaterials: 'materials'),
+  // but we reference the keys several times in this component, we want to return the
+  // right key for the path we are currently on
+  return (
+    Object.entries(TEACHER_NAVIGATION_PATHS).find(
+      ([, val]) => val === urlPath
+    )?.[0] || ''
+  );
+}
+
```

Comment:
Note: I initially tried to see if `useParams` would be applicable here, but as far as I can tell, because the paths are setup with strings (ex. `/progress`) instead of a dynamic route (like `:section_id` but for `:path`), it seems like it seems like `useParams` is not the right solution here.  In the prior PR @lfryemason mentioned working on something related.  It is possible that this might not be the right solution once his work is done.